pkg/kubelet/cri/remote/remote_image.go

```diff
@@ -226,6 +226,16 @@ func (r *remoteImageService) ImageFsInfo(ctx context.Context) ([]*runtimeapi.Fil
 	return r.imageFsInfoV1(ctx)
 }
 
+// ImageFsInfo returns information of the filesystem that is used to store images.
+func (r *remoteImageService) ImageFsInfoV2(ctx context.Context) (*runtimeapi.ImageFsInfoResponse, error) {
+	// Do not set timeout, because `ImageFsInfo` takes time.
+	// TODO(random-liu): Should we assume runtime should cache the result, and set timeout here?
```

Comment:
I copied these TODO from the previous one but I don't really know if these will ever be valid.  Left them in for now.